modules/base/rendering/screenspacedate.cpp

```diff
@@ -0,0 +1,108 @@
+#include <openspace/util/time.h>
+#include <openspace/util/timemanager.h>
+#include <ghoul/logging/logmanager.h>
+#include <ghoul/opengl/framebufferobject.h>
+#include <ghoul/opengl/texture.h>
+
+namespace {
+    constexpr openspace::properties::Property::PropertyInfo FormatStringInfo = {
+        "FormatString",
+        "Format string",
+        "The format text describing how this dashboard item renders its text. This text "
+        "must contain exactly one {} which is a placeholder that will contain the date "
+        "in the format as specified by `TimeFormat`.",
+        openspace::properties::Property::Visibility::AdvancedUser
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo TimeFormatInfo = {
+        "TimeFormat",
+        "Time format",
+        "The format string used for formatting the date/time before being passed to the "
+        "string in FormatString. See "
+        "https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/C/cspice/timout_c.html for full "
+        "information about how to structure this format.",
+        openspace::properties::Property::Visibility::User
+    };
+
+    // This `ScreenSpaceRenderable` shows the current in-game simulation time. The
+    // `FormatString` and the `TimeFormat` options provide the ability to customize the
+    // output that is printed. See these two parameters for more information on how to
+    // structure the inputs.
+    struct [[codegen::Dictionary(ScreenSpaceTextDate)]] Parameters {
+        // [[codegen::verbatim(FormatStringInfo.description)]]
+        std::optional<std::string> formatString;
+
+        // [[codegen::verbatim(TimeFormatInfo.description)]]
+        std::optional<std::string> timeFormat;
+    };
+#include "screenspacedate_codegen.cpp"
+} // namespace
+
+namespace openspace {
+
+documentation::Documentation ScreenSpaceDate::Documentation() {
+    return codegen::doc<Parameters>(
+        "base_screenspace_date",
+        ScreenSpaceRenderableText::Documentation()
+    );
+}
+
+ScreenSpaceDate::ScreenSpaceDate(const ghoul::Dictionary& dictionary)
+    : ScreenSpaceRenderableText(dictionary)
+    , _formatString(FormatStringInfo, "Date: {}")
+    , _timeFormat(TimeFormatInfo, "YYYY MON DD HR:MN:SC.### UTC ::RND")
+{
+    const Parameters p = codegen::bake<Parameters>(dictionary);
+
+    _formatString = p.formatString.value_or(_formatString);
+    addProperty(_formatString);
+
+    _timeFormat = p.timeFormat.value_or(_timeFormat);
+    addProperty(_timeFormat);
+}
+
+void ScreenSpaceDate::update() {
+    std::string time = SpiceManager::ref().dateFromEphemerisTime(
+        global::timeManager->time().j2000Seconds(),
+        _timeFormat.value().c_str()
+    );
+
+    try {
+        // @CPP26(abock): This can be replaced with std::runtime_format
+        _buffer = std::vformat(_formatString.value(), std::make_format_args(time));
+    }
+    catch (const std::format_error&) {
+        LERRORC("DashboardItemDate", "Illegal format string");
```

Comment:
Wrong category
```suggestion
        LERRORC("ScreenSpaceDate", "Illegal format string");
```

or maybe add a `_loggerCat`? Easier to keep in sync when more log messages are added